src/webgl/material.js

```diff
@@ -531,11 +531,17 @@ p5.prototype.textureWrap = function(wrapX, wrapY = wrapX) {
 };
 
 /**
- * Normal material for geometry is a material that is not affected by light.
- * It is not reflective and is a placeholder material often used for debugging.
- * Surfaces facing the X-axis, become red, those facing the Y-axis, become green and those facing the Z-axis, become blue.
- * You can view all possible materials in this
+ * Creates a normal material.
```

Comment:
One question, how do you feel about the verb 'creates' across the board? I feel this implies that an object is returned that can be used elsewhere. Perhaps 'Sets the current material to' is more accurate? Open to other ideas ...